spec/app/domain/util/trackable_error_spec.rb

```diff
@@ -21,3 +21,33 @@
     end
   end
 end
+
+require 'util/error_code'
+
+describe Error::ConjurCode do
+  let(:error_code_valid) { Error::ConjurCode.new('./spec/app/domain/util/errors/valid') }
+  let(:error_code_invalid) { Error::ConjurCode.new('invalid_file_path') }
+  let(:error_code_empty) { Error::ConjurCode.new('./spec/app/domain/util/errors/empty') }
+  
+  it 'raises error when file path is invalid' do
+    expect { error_code_invalid }.to raise_error(RuntimeError)
+  end
+
+  it 'outputs the next available error code' do 
+    expect do
+      error_code_valid.print_next_available
+    end.to output("The next available error number is 5 ( CONJ00005E )\n").to_stdout
+  end
+
+  it 'outputs next available error code but forgets to append leading zeros' do 
+    expect do
+      error_code_valid.print_next_available
+    end.to_not output("The next available error number is 58 ( CONJ5E )\n").to_stdout
+  end
```

Comment:
This result of this test is logically implied by the result of the test above, so it can be removed.